getting-started/2/README.md

```diff
@@ -298,34 +298,31 @@ But, with a much better querying experience.
 ## Deleting a node
 
 Ratel also makes it easier to compose mutation to delete a node.
+Just click on the node and then click the delete button.
 
-Just click on the node and then click the delete button. 
-In the text area, you should now see a mutation to delete the node.
+In the text area, you should now see a mutation to delete the node as follow.
```

Comment:
Since the mutation will already be there in the text area. I don't see a need for `as follow`.